packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -153,6 +153,8 @@ export function generateGCConfigs(
 		throw new UsageError("inactive timeout should not be greater than the sweep timeout");
 	}
 
+	const throwOnInactiveLoad: boolean | undefined = createParams.gcOptions.throwOnInactiveLoad;
```

Comment:
nit: Make this a `boolean` by setting it to false if the option is undefined.

Reply:
Good point, but I'm just going to click Merge on my phone and carry on with my time off for now :)